requirements.txt

```diff
@@ -0,0 +1,4 @@
+Flask==0.10
+icalendar==3.8
+mock==1.0
+python-dateutil==2.1
```

Comment:
I'll add requests, and yes, I'm not sure why pip picked 0.10. I'll update the versions.